Makefile

```diff
@@ -27,4 +27,11 @@ ios-install:
 
 ##@ Tests
 test:
-	npm test
\ No newline at end of file
+	npm test
+
+##@ Lint
+lint-check:
+	npm run lint
+
+lint-fix:
```

Comment:
Si cette ligne est utilisé par un humain ajouter une description du style
lint-fix: ## runs ESLint to fix the code according to the defined rules.
pourrais permettre de la voir quand on tape make help